package.json

```diff
@@ -52,7 +52,9 @@
     "dev": "concurrently --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"http-server -c-1 -p 8080\"",
     "dev-test": "concurrently --names \"ROLLUP,ROLLUPTEST,HTTP\" -c \"bgBlue.bold,bgRed.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"rollup -c test/rollup.unit.config.js -w -m inline\" \"http-server -p 8080\"",
     "start": "npm run dev",
-    "lint": "eslint src --ext js --ext ts && tsc -p utils/build/tsconfig.lint.json",
+    "lint": "npm run lint-source && npm run lint-examples",
```

Comment:
How about writing the commands like so:
```
"lint": "npm run lint-source && npm run lint-examples && npm run lint-typescript",
"lint-source": "eslint src --ext js --ext ts",
"lint-examples": "eslint examples/jsm --ext js --ext ts --ignore-pattern libs",
"lint-typescript": "tsc -p utils/build/tsconfig.lint.json"
```
IMO, the commands are more clear in this way.

Reply:
I agree! Just updated